.cirrus.yml

```diff
@@ -46,15 +45,3 @@ linux_aarch64_task:
 #    - refreshenv
 #    - echo PATH=%PATH% >> "%CIRRUS_ENV%"
 #  <<: *BUILD_AND_STORE_WHEELS
-
-macos_arm64_task:
```

Comment:
why did the arm64 mac wheels get dropped from cirrus? I did try the universal-only option and found that the cross-compiled wheels didn't get picked up by M1/M2 macs.. so I got confused by what really happens. Maybe this is scikit-build legacy issue and it was fixed in scikit-build-core?

Reply:
Ahh, I assumed since you were making universal before that they worked. :) It's quite possible it's not cross compiling. Probably can be fixed (cross compiles are pretty easy on macOS), but maybe you really just want x86 and arm without a universal. (You can merge them as well if you want, but two wheels is fine for most use cases).

Reply:
Pretty sure autotools do not support cross compiling. Probably can be hacked.